### PR TITLE
release version 0.4.4

### PR DESCRIPTION
**Pull request by jor-**
I would like to release version 0.4.4. It fixes the bug in `solve`, where the `base` attribute of the result array is not set correctly. Besides this the tests are now using conda.

### CODE REVIEW

File: sksparse/cholmod.pyx [hunk 1]
```diff
@@ -287,77 +287,94 @@ cdef int _np_typenum_for_index(int itype):
...
+    # convert cholmod array to numpy array
+    cdef np.ndarray array = np.PyArray_SimpleNewFromData(
+        1, [m.ncol * m.nrow], _np_typenum_for_data(m.xtype), m.x)
+    # set destructor that frees the memory as base
+    np.set_array_base(array, base)
+    # reshape array
+    array = array.reshape((m.ncol, m.nrow)).T
+    # check if array is writeable
+    assert np.PyArray_ISWRITEABLE(array)
+    # return array
```
Comment (anntzer) [inline sksparse/cholmod.pyx]: remove this commented line?

File: sksparse/cholmod.pyx [hunk 2]
```diff
@@ -287,77 +287,94 @@ cdef int _np_typenum_for_index(int itype):
...
-    PyArray_ENABLEFLAGS(indices, np.NPY_WRITEABLE)
-    np.set_array_base(indices, cleaner)
     cdef np.ndarray data = np.PyArray_SimpleNewFromData(
         1, [m.nzmax], _np_typenum_for_data(m.xtype), m.x)
-    PyArray_ENABLEFLAGS(data, np.NPY_WRITEABLE)
-    np.set_array_base(data, cleaner)
 
+    # set destructor and check if writeable
+    for array in (indptr, indices, data):
+        np.set_array_base(array, base)
```
Comment (anntzer) [inline sksparse/cholmod.pyx]: I don't understand why this call works but the previous version didn't.

File: sksparse/cholmod.pyx [hunk 3]
```diff
@@ -287,77 +287,94 @@ cdef int _np_typenum_for_index(int itype):
...
-    # Construct cleaner first, so even if we later raise an exception we still
-    # fulfill the contract that we will take care of cleanup:
-    cdef _DenseCleanup cleaner = _DenseCleanup()
-    cleaner._dense = m
-    cleaner._common = common
-
-    cdef np.ndarray out = np.PyArray_SimpleNewFromData(
-        1, [m.ncol * m.nrow], _np_typenum_for_data(m.xtype), m.x).reshape((m.ncol, m.nrow)).T
-    PyArray_ENABLEFLAGS(out, np.NPY_WRITEABLE)
-    np.set_array_base(out, cleaner)
```
Comment (jor-) [inline sksparse/cholmod.pyx]: The bug was here or to be precise two lines above. Due to the `reshape` and `transpose` two lines above, the `out` array was not anymore the array constructed from the Cholmod data. This is the reason why the `base` attribute was already set.

**Comment by jor-** [comment]
Thanks for reviewing. I am going to make the release.

**Pull request merged**